packages/synapse-sdk/src/telemetry/get-sentry.ts

```diff
@@ -0,0 +1,27 @@
+import type * as SentryBrowser from '@sentry/browser'
+import type * as SentryNode from '@sentry/node'
+
+// Dynamically import the correct Sentry package based on environment
+const isBrowser =
+  typeof (globalThis as any).window !== 'undefined' && typeof (globalThis as any).document !== 'undefined'
+
+export type Sentry = typeof SentryNode.default | typeof SentryBrowser.default
+
+export async function getSentry(): Promise<{ Sentry: Sentry; integrations: any[] }> {
```

Comment:
Somewhere we want to callout that we're adding additional integrations to the defaults that Sentry enables per https://docs.sentry.io/platforms/javascript/configuration/integrations/